tests/playbooks/test_minimal.yaml

```diff
@@ -2,79 +2,79 @@
   import_playbook: _before_adoption.yaml
 
 - name: Adoption
-  hosts: localhost
+  hosts: "{{ adoption_host | default('localhost') }}"
```

Comment:
This line is good but the rest of the changes in the file should go away?

Reply:
right  (facepalm:), patching